configs/coins/bsc_archive.json

```diff
@@ -0,0 +1,74 @@
+{
+  "coin": {
+    "name": "BNB Smart Chain Archive",
+    "shortcut": "BNB",
+    "label": "BNB Smart Chain",
+    "alias": "bsc_archive"
+  },
+  "ports": {
+    "backend_rpc": 8065,
+    "backend_p2p": 38365,
+    "backend_http": 8165,
+    "blockbook_internal": 9065,
+    "blockbook_public": 9165
+  },
+  "ipc": {
+    "rpc_url_template": "ws://127.0.0.1:{{.Ports.BackendRPC}}",
+    "rpc_timeout": 25
+  },
+  "backend": {
+    "package_name": "backend-bsc-archive",
+    "package_revision": "satoshilabs-1",
+    "system_user": "bsc",
+    "version": "1.1.19",
+    "binary_url": "https://github.com/bnb-chain/bsc/releases/download/v1.1.19/geth_linux",
+    "verification_type": "sha256",
+    "verification_source": "ad49b00a5df44fb67e0dc166a7cf02a02c67ab296d25e4ac3d56134567577c41",
+    "extract_command": "mv ${ARCHIVE} backend/geth_linux && chmod +x backend/geth_linux && echo",
+    "exclude_files": [],
+    "exec_command_template": "/bin/sh -c '{{.Env.BackendInstallPath}}/{{.Coin.Alias}}/bsc_exec.sh 2>> {{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend/{{.Coin.Alias}}.log'",
+    "exec_script": "bsc.sh",
```

Comment:
Hi, I just built the archive backend. Unfortunately the config seems to me a bit messed up, fortunately in a way that the backend did not start the non archive version :D 

When starting the service I got the error `/bin/sh: 1: /opt/coins/nodes/bsc_archive/bsc_exec.sh: not found`. In the folder `/opt/coins/nodes/bsc_archive/` there is file `bsc_archive_exec.sh`, which contains the non archive script :D

I think the `exec_command_template` should start `bsc_exec_archive.sh` instead of `bsc_exec.sh` and the `exec_script` should be `bsc_archive.sh` instead of `bsc.sh`.

Reply:
Yep, this works:
```
    "exec_command_template": "/bin/sh -c '{{.Env.BackendInstallPath}}/{{.Coin.Alias}}/bsc_archive_exec.sh 2>> {{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend/{{.Coin.Alias}}.log'",
    "exec_script": "bsc_archive.sh",
```

Reply:
Man I really botched the copy of the archive config... Sorry for all the issues there. Just pushed up that change

Reply:
Don't worry, it just seemed funny to me that two independent bugs averted potentially larger issue for me (I do not know what would happen if I started the non archive backend to my archive DB).
And in general, the exec script functionality works really well, good work!

Reply:
Ha yeah for real. Glad your archive db remained safe, that would have sucked.. Glad the exec scripts are working, and I assume the post install worked even though you aren't using the genesis file since you restored from a snapshot?